RunGcm-filters-leticia.ipynb

```diff
@@ -0,0 +1,3088 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #idk what chunk is...
</pre><p>After Deepak's dask tutorial today, you probably know now! :) This cell tells dask the chunk size in nlon and nlat dimension.</p><p> </p><p>You could enter a new cell where you just type</p><pre class="ql-syntax" spellcheck="false">ssh
</pre><p>so it becomes clear that we are working with dask arrays - pretty cool for the presentation tomorrow!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw22-proj-gcmfilters/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>